### DISCUSSION TITLE
Car Settings not working

### DISCUSSION THREAD

**Discussion by 0barning** [tillsteinbach/VWsFriend#560] [General]
I've just installed VWsFriend and connected it to my We-Connect account. Data is coming in, but I cannot access my Car Settings dashboard, or Geo Locations. I got the message 'Page not Found', do I need to do additional things to get it to work?

**Comment by tillsteinbach** [comment]
Are the connection settings at http://YourBackendIP:4000/database/settings/edit correct?

**Reply by 0barning** [reply]
No, I adjusted it. Now it's working. I used different ports because of a running Teslamate instance on the same host

**Reply by tillsteinbach** [reply]
Perfect!
